### PR TITLE
New Chaplain Weapon: Magic Shard!

### PR DESCRIPTION
**Pull request by ghost**
Works just like the null rod until you use it. Using it changes your race and makes your weapon do less damage (18>10). You can only get one random race roll so don't screw up!

It'll be fun for people who want to roll a cool race but they'll still be able to do chaplainy things with their shard, just less damage.

### CODE REVIEW

**Comment by Armhulen** [comment]
I'm gonna finish this as soon as i'm able to!

**Comment by Armhulen** [comment]
confused hit and run

**Comment by DoctoreMad** [comment]
Man,
What?

**Comment by KorPhaeron** [comment]
What is the end game here I don't understand

**Comment by Armhulen** [comment]
The end game is cool items that aren't x but different skin, my last two changes have you change your chaplain fighting style

**Comment by KorPhaeron** [comment]
This is starting to cross the line from silly fluff weapons to becoming wizard powers lite though

**Comment by DoctoreMad** [comment]
In my opinion, this PR should be a bit more controversial. It smells like pure salt will come form people simply rolling a shit race or overpowered race with no in between.
Roundstart suicide due to not getting a good race with Magic Shard when?

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -319,6 +319,25 @@
...
+	//icons soon
+	force = 0
+	throwforce = 4
+	
+/obj/item/nullrod/crystal/magicshit
+	var/reagent_amount = 1
+	var/reagent_type = "unstablemutationtoxin"
+
+/obj/item/nullrod/crystal/attack_self(/mob/living/carbon/human/user)
+	var/crush = "<span class='danger'>you crush the crystal in your hand and a torrent of energy rushes into you!</span>"
```
Comment (ShizCalev) [inline code/game/objects/items/holy_weapons.dm]: Why is this a var?

File: code/game/objects/items/holy_weapons.dm [hunk 2]
```diff
@@ -319,6 +319,25 @@
...
+	to_chat(user, crush)
```
Comment (ShizCalev) [inline code/game/objects/items/holy_weapons.dm]: `to_chat(user, "<span class='danger'>you crush the crystal in your hand and a torrent of energy rushes into you!</span>")` instead

File: code/game/objects/items/holy_weapons.dm [hunk 3]
```diff
@@ -319,6 +319,25 @@
...
+		user.add_reagent(reagent_type, reagent_amount)
```
Comment (ShizCalev) [inline code/game/objects/items/holy_weapons.dm]: No check to see if it failed or succeeded

File: code/game/objects/items/holy_weapons.dm [hunk 4]
```diff
@@ -319,6 +319,25 @@
...
+		return TRUE
+	var/fail = "<span class='danger'>I guess the energy didn't work?</span>"
```
Comment (ShizCalev) [inline code/game/objects/items/holy_weapons.dm]: Is this doing anything?

**Review by ShizCalev — CHANGES_REQUESTED** [review]
What's even going on here?

File: code/game/objects/items/holy_weapons.dm (hunk 3 above)
Comment (Iamgoofball) [reply code/game/objects/items/holy_weapons.dm]: why is this tabbed in

**Comment by Repukan** [comment]
Chaplain is supposed to be a gimmick role out of the volition of the player, not the features that are powercrept into the role.

File: code/game/objects/items/holy_weapons.dm (hunk 2 above)
Comment (Armhulen) [reply code/game/objects/items/holy_weapons.dm]: Okay

**Comment by Armhulen** [comment]
>10 damage weapon

>powercreep

??

**Comment by Repukan** [comment]
Are we acting stupid now and ignoring the mutations

**Comment by Bawhoppen** [comment]
have we lost our way?

**Comment by Armhulen** [comment]
Your shittier null rod MUST be your weapon to deflect cult magic, I could nerf it more if you'd like

**Comment by Armhulen** [comment]
Hell let's give them a 1 force shard but the mutation so they use the shard for tune banishment and cult protection and the race for fighting. Would that be better

File: code/game/objects/items/holy_weapons.dm (hunk 4 above)
Comment (Armhulen) [reply code/game/objects/items/holy_weapons.dm]: Whoops

File: code/game/objects/items/holy_weapons.dm (hunk 3 above)
Comment (Armhulen) [reply code/game/objects/items/holy_weapons.dm]: I had an if statement I'll fix it

**Comment by Armhulen** [comment]
>having nonhuman chaplains

wait that's heresy

**Comment by Armhulen** [comment]
>having nonhuman chaplains

wait that's heresy

**Pull request closed without merging**